### PR TITLE
Fire pixel on https upgrade failures

### PR DESCRIPTION
**Pull request by subsymbolic**
Task/Issue URL: https://app.asana.com/0/361428290920652/832602107339144/f
Tech Design URL: N/A however spec was discussed in https://app.asana.com/0/56420494107937/832602107339145/f

**Description**:
Fire pixel on https upgrade failures to allow bad data to be removed from lists

**Steps to test this PR**:
1. Go to a broken https upgrade site ensure that the new ehd pixel is fired.
It may be easier to got to `HTTPSUpgrade.swift` and return true from the ` isInUpgradeList` method. Then visit https://badssl.com and tap on the "expired" link.

1. Ensure that existing pixels fire as normal

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)

### CODE REVIEW

File: Core/URLExtension.swift [hunk 1]
```diff
@@ -33,6 +33,16 @@ extension URL {
     enum Host: String {
         case localhost
     }
+    
+    public var simpleUrl: String {
```
Comment (brindy) [inline Core/URLExtension.swift:37]: (Optional) Not keen on this name but I'm struggling to suggest something better.  I don't like the `Url` part (see `absoluteString`) and it feels like it's doing something quite specific, "simple" being subjective.

File: DuckDuckGoTests/URLExtensionTests.swift [hunk 2]
```diff
@@ -20,11 +20,36 @@
...
+        let url = "http://subdomain.example.com"
+        XCTAssertEqual(url, URL(string: url)!.simpleUrl)
+    }
+    
+    func testWhenUriContainsPathThenSimpleUrlIsEqual() {
+        let url = "http://example.com/about"
+        XCTAssertEqual(url, URL(string: url)!.simpleUrl)
+    }
+    
+    func whenUriContainsUsernameThenSimpleUrlOmitsThis() {
```
Comment (brindy) [inline DuckDuckGoTests/URLExtensionTests.swift]: Missing `test` prefix.

File: DuckDuckGoTests/URLExtensionTests.swift [hunk 3]
```diff
@@ -20,11 +20,36 @@
 import XCTest
 
 class URLExtensionTests: XCTestCase {
```
Comment (brindy) [inline DuckDuckGoTests/URLExtensionTests.swift:22]: Doesn't appear to be any test cases that don't use a scheme (even though the code deals with that scenario).

**Review by brindy — COMMENTED** [review]
Couple of minor things and one optional thing. 👍 

Feel free to merge once you've addressed them.

**Review by brindy — APPROVED** [review]
LGTM!

**Comment by subsymbolic** [comment]
This was merged as a hotfix.

**Pull request closed without merging**